app/views/unattended/provisioning_templates/host_init_config/host_init_config_default.erb

```diff
@@ -54,6 +58,16 @@ trap 'exit_and_cancel_build' ERR
 
 <% end -%>
 
+# Subscription manager - refresh subscription data
+
+<% if plugin_present?('katello') -%>
+if [ x$ID = xrhel ] || [ x$ID = xcentos ] || [ x$ID = xol ]; then
```

Comment:
should we instead add `@host.operatingsystem.family == "RedHat` to trully support all OSes listed in the metadata? it could be added to the `plugin_present?` condition perhaps